### PR TITLE
cfn-flip 1.2.3 (new formula)

### PR DESCRIPTION
**Pull request by stilvoid**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/cfn-flip.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "Properties": {
+              "BucketName": {
+                "Ref": "AWS::StackName"
+              }
+            }
+          }
+        }
+      }
+    EOS
+    system bin/"cfn-flip", "test.json"
```
Comment (chenrui333) [inline Formula/cfn-flip.rb:46]: can you assert the response? Thanks!

**Comment by chenrui333** [comment]
lgtm other than just one nitpick comment.

File: Formula/cfn-flip.rb (hunk 1 above)
Comment (stilvoid) [reply Formula/cfn-flip.rb]: Good call - done :+1:

**Review by dtrodrigues — APPROVED** [review]
Thanks for your contribution @stilvoid

**Pull request closed without merging**